### ISSUE TITLE
Issue using vbuild as a local dependency

### ISSUE THREAD

**Issue by zephraph** [egoist/vbuild#16]
I first installed vbuild on Friday. The setup I have currently worked at that time. After updating to the latest (2.2.0) my build is now failing. 

It looks like it's due to failing to find items internally in vbuild... seems strange to me. Has anyone noticed anything similar or know what may be causing the issue?

```
> vue-prototype@1.0.0 dev /Users/<user>/Git/vue-prototype
> vbuild --dev --lint

> vbuild is running at http://localhost:4000

> Mode      Hot Reloading, IIFE
> Entry[0]  src/index.js
> Entry[1]  ~/Git/vue-prototype/node_modules/vbuild/node_modules/webpack-hot-middleware/client
> Output    ~/Git/vue-prototype/dist/[name].js
> Config    ~/Git/vue-prototype/vue.config.json
> Port      4000
> Devtool   eval-source-map

webpack: wait until bundle finished: /
webpack built 1c75320386ad96fb425c in 985ms
Hash: 1c75320386ad96fb425c
Version: webpack 2.1.0-beta.22
Time: 985ms
    Asset     Size  Chunks       Chunk Names
client.js  28.6 kB       0       client

ERROR in multi client
Module not found: Error: Can't resolve '/Users/<user>/Git/vue-prototype/node_modules/vbuild/node_modules/webpack-hot-middleware/client' in '/Users/<user>/Git/vue-prototype'
 @ multi client

ERROR in ./src/index.js
Module build failed: Error: Cannot read config file: /Users/<user>/Git/vue-prototype/node_modules/vbuild/node_modules/eslint-config-egoist-vue/index.js
Error: Cannot find module '/Users/<user>/Git/vue-prototype/node_modules/vbuild/node_modules/eslint-config-egoist-vue/index.js'
    at Function.Module._resolveFilename (module.js:339:15)
    at module.exports (/Users/<user>/Git/vue-prototype/node_modules/resolve-from/index.js:14:16)
    at module.exports (/Users/<user>/Git/vue-prototype/node_modules/require-uncached/index.js:11:17)
    at loadJSConfigFile (/Users/<user>/Git/vue-prototype/node_modules/eslint/lib/config/config-file.js:160:16)
    at loadConfigFile (/Users/<user>/Git/vue-prototype/node_modules/eslint/lib/config/config-file.js:199:22)
    at Object.load (/Users/<user>/Git/vue-prototype/node_modules/eslint/lib/config/config-file.js:498:18)
    at loadConfig (/Users/<user>/Git/vue-prototype/node_modules/eslint/lib/config.js:63:33)
    at new Config (/Users/<user>/Git/vue-prototype/node_modules/eslint/lib/config.js:228:38)
    at CLIEngine.executeOnText (/Users/<user>/Git/vue-prototype/node_modules/eslint/lib/cli-engine.js:736:28)
    at lint (/Users/<user>/Git/vue-prototype/node_modules/eslint-loader/index.js:42:18)
    at Object.module.exports (/Users/<user>/Git/vue-prototype/node_modules/eslint-loader/index.js:166:3)
 @ multi client
webpack: bundle is now VALID.
```

**Comment by zephraph** [comment]
Oddly enough, going into the vbuild directory and issuing `npm install` fixes the issue. It looks like certain items are expecting an absolute path to exist when it may not.

**Comment by egoist** [comment]
OK, when you install vbuild as devDependency I guess `webpack-hot-middleware` is installed under `your-project/node_modules/webpack-hot-middleware` instead of `your-project/node_modules/vbuild/node_modules/webpack-hot-middleware`.

And we use `__dirname` of vbuild to resolve `webpack-hot-middleware`, that's why this issue occurs, will fix it.

**Comment by neves** [comment]
This problem persist with vbuild 3.1.4
reproducible code:
```
npm i -g vbuild
vbuild init bug
cd bug
npm i -D vbuild
npm i
npm run dev
...
ERROR in multi client
Module not found: Error: Can't resolve '/.../bug/node_modules/vbuild/node_modules/webpack-hot-middleware/client' in '/.../bug'
 @ multi client

# using vbuild --dev globally works fine
vbuild --dev
```

**Comment by zephraph** [comment]
Must be a new thing. It was resolved. @egoist is probably in bed, I'll take a look at it soon.

**Comment by egoist** [comment]
@neves fixed in `v3.1.5`
